code/modules/projectiles/special.dm

```diff
@@ -212,6 +212,31 @@ ABSTRACT_TYPE(/datum/projectile/special)
 		FC.travelled = rand(0,dissipation_variance)
 		FC.launch()
 
+/datum/projectile/special/spreader/buckshot_burst/glass
+	spread_projectile_type = /datum/projectile/bullet/improvglass
+	name = "glass"
+	sname = "glass"
+	cost = 1
+	pellets_to_fire = 6
+	shot_sound = 'sound/weapons/shotgunshot.ogg'
+	speed_max = 20
+	speed_min = 40
+	spread_angle_variance = 30
+	dissipation_variance = 40
+
+/datum/projectile/special/spreader/buckshot_burst/scrap
+	name = "fragments"
+	sname = "fragments"
+	cost = 1
+	pellets_to_fire = 3
+	spread_projectile_type = /datum/projectile/bullet/improvscrap
+	shot_sound = 'sound/weapons/shotgunshot.ogg'
+	speed_max = 20
+	speed_min = 60
```

Comment:
2 things:

- You have a higher min than max
- too much variance here tends to feel really janky - I've found that a min of 28 and a max of 36 or so is about as far as I could go without it looking weird.

Ditto above for glass projectile